### PR TITLE
libspirv.h in SPIRV-Tools removed macros for version and revision.

### PR DESCRIPTION
**Pull request by antiagainst**
Counterpart for https://github.com/KhronosGroup/SPIRV-Tools/pull/170

### CODE REVIEW

File: third_party/Android.mk [hunk 1]
```diff
@@ -107,11 +107,12 @@ LOCAL_SRC_FILES:= \
 		source/operand.cpp \
 		source/print.cpp \
 		source/spirv_endian.cpp \
+		source/spirv_target_env.cpp \
```
Comment (dekimir) [inline third_party/Android.mk]: This file should go into a separate commit with a different title.

Comment (dekimir) [inline third_party/Android.mk]: Clarification: by "this file", I mean `third_party/Android.mk`.

Comment (antiagainst) [inline third_party/Android.mk]: Done.

**Comment by dekimir** [comment]
+2 but pls don't squash
